Nos/Controller/NoteEditorController.swift

```diff
@@ -2,7 +2,7 @@ import Foundation
 import SwiftUI
 import UIKit
 
-/// A controller for Nostr note text that is being edited. This controllers pairs with a `NoteUITextViewRepresentable`
+/// A controller for Nostr note text that is being edited. This controller pairs with a `NoteUITextViewRepresentable`
```

Comment:
Oh, did I forget to post this? I found that we _can_ do the double-backtick thing for NoteUITextViewRepresentable if we do this:

```suggestion
/// A controller for Nostr note text that is being edited. This controller pairs with a ``NoteTextEditor/NoteUITextViewRepresentable``
```
...which then makes the line too long so we'll need to break it, but that works for me to magically link to the struct. Ditto below.